pkg/config/read_test.go

```diff
@@ -81,16 +97,18 @@ func WSServer(ctx context.Context, data map[string]interface{}) {
 			}
 
 			writer, _ := conn.NextWriter(websocket.BinaryMessage)
-			json.NewEncoder(writer).Encode(data)
+			_ = json.NewEncoder(writer).Encode(data)
 		}
 	})
 
 	s.Handler = m
 
-	go s.ListenAndServe()
+	go func() {
```

Comment:
this is a bit strict, especially for tests. Maybe we should skip the ci analysis for test files...

Reply:
yuck, well..